mmv1/products/spanner/Instance.yaml

```diff
@@ -261,6 +258,16 @@ properties:
       - 'STANDARD'
       - 'ENTERPRISE'
       - 'ENTERPRISE_PLUS'
+  - name: 'instanceType'
+    type: Enum
+    description: |
```

Comment:
Can you add the restrictions on not setting `processing_units` and the other values here? Moving that to the description will increase visibility if we remove the validation